### PR TITLE
Replace domain aliases with luadns templates and clones

### PR DESCRIPTION
**Pull request by J0WI**
This merges some equal domain zones. Please verify that the syntax is correct before merging. You may temporary switch to a shorter [user TTL](http://www.luadns.com/help.html#user-default-ttl) for testing.

### CODE REVIEW

File: pirateparty.ch.lua [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+a("www", "149.126.4.43")
+a("archive", "149.126.4.43")
+a("mail", "149.126.4.43")
+a("lists", "149.126.4.43")
+
+mx(_a, "piratenpartei.ch", 10)
+mx("www", "piratenpartei.ch", 10)
+mx("archive", "piratenpartei.ch", 10)
+mx("mail", "piratenpartei.ch", 10)
+mx("lists", "piratenpartei.ch", 10)
```
Comment (philhug) [inline pirateparty.ch.lua]: why do you add MX for things like www, archive, mail, lists?
MX is only used for main domain and lists

File: templates/sections.lua [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  mx("tg", target, 10)
+  mx("ti", target, 10)
+  mx("ur", target, 10)
+  mx("vd", target, 10)
+  mx("vs", target, 10)
+  mx("winterthur.zh", target, 10)
+  mx("zg", target, 10)
+  mx("zh", target, 10)
+  mx("zs", target, 10)
+end
```
Comment (philhug) [inline templates/sections.lua]: did we really have mail enabled on all variants?
IMO it was only enabled for the domains without dashes.

Comment (J0WI) [reply templates/sections.lua]: Nope, they are all configured, even with dashes.

File: pirateparty.ch.lua (hunk 1 above)
Comment (J0WI) [reply pirateparty.ch.lua]: `lists` is in use, `mail` is deprecated. `www` and `archive` are not in use.

Comment (cambid) [reply pirateparty.ch.lua]: @J0WI: Can you remove the mx-records for www and archive and keep the other two please?

File: templates/sections.lua (hunk 2 above)
Comment (cambid) [reply templates/sections.lua]: Ok, then let's keep them.

File: pirateparty.ch.lua (hunk 1 above)
Comment (J0WI) [reply pirateparty.ch.lua]: @cambid done.

File: pirateparty.ch.lua [hunk 3]
```diff
@@ -0,0 +1,13 @@
+txt(_a, "v=spf1 a mx ?all")
+
+a(_a, "149.126.4.43")
+a("www", "149.126.4.43")
+a("archive", "149.126.4.43")
+a("mail", "149.126.4.43")
+a("lists", "149.126.4.43")
+
+mx(_a, "piratenpartei.ch", 10)
+mx("mail", "piratenpartei.ch", 10)
```
Comment (hug-abacus) [inline pirateparty.ch.lua]: Why do we need an MX for mail?

File: pirateparty.ch.lua [hunk 4]
```diff
@@ -0,0 +1,13 @@
+txt(_a, "v=spf1 a mx ?all")
```
Comment (hug-abacus) [inline pirateparty.ch.lua]: This changes current behaviour, please use a separate PR for this.

Comment (J0WI) [reply pirateparty.ch.lua]: Which host would be affected by this?
Existing aliases already have this:
https://github.com/ppschweiz/domains/blob/6a19e3815a7592b7b88c445d5caf1684b2f45446/parti-pirate.ch.bind#L4
The only exception is `piratenpartei.ch` that is not in the clones file and therefore not changed.

**Comment by J0WI** [comment]
We are already using lua templates btw: https://github.com/ppschweiz/domains/search?l=lua

**Pull request merged**